service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -158,20 +159,26 @@ void aboutToSubmitShouldPopulateCaseManagementOrder() throws Exception {
         Map<String, Object> data = new HashMap<>();
 
         Stream.of(DirectionAssignee.values()).forEach(direction ->
-            data.put(direction.getValue() + "Custom", createElementCollection(createUnassignedDirection()))
+            data.put(direction.toCustomDirectionField(), createElementCollection(createUnassignedDirection()))
         );
 
         data.put("cmoHearingDateList", dynamicHearingDates);
-        data.put("caseManagementOrder", CaseManagementOrder.builder().status(SELF_REVIEW).build());
+        data.put("caseManagementOrder", CaseManagementOrder.builder()
+            .orderDoc(DocumentReference.builder().filename("draft-case-management-order.pdf").build())
+            .status(SELF_REVIEW)
+            .action(OrderAction.builder().changeRequestedByJudge("Changes").build())
+            .build());
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = getResponse(data, "about-to-submit");
         CaseData caseData = mapper.convertValue(callbackResponse.getData(), CaseData.class);
         CaseManagementOrder caseManagementOrder = caseData.getCaseManagementOrder();
 
         assertThat(caseManagementOrder.getDirections()).containsAll(createCmoDirections());
-        assertThat(caseManagementOrder).extracting("id", "hearingDate")
-            .containsExactly(fromString("b15eb00f-e151-47f2-8e5f-374cc6fc2657"), TODAYS_DATE.plusDays(5).toString());
+        assertThat(caseManagementOrder.getId()).isEqualTo(fromString("b15eb00f-e151-47f2-8e5f-374cc6fc2657"));
+        assertThat(caseManagementOrder.getHearingDate()).isEqualTo(TODAYS_DATE.plusDays(5).toString());
         assertThat(caseManagementOrder.getStatus()).isEqualTo(SELF_REVIEW);
+        assertThat(caseManagementOrder.getOrderDoc().getFilename()).isEqualTo("draft-case-management-order.pdf");
+        assertThat(caseManagementOrder.getAction().getChangeRequestedByJudge()).isEqualTo("Changes");
     }
 
     //TODO: caseDetails before is in this test as a start for conditional call to submitted code.
```

Comment:
To be honest I don't get that TODO :D

Reply:
I think that todo was from when we previously had the draft cmo controller have some logic in it so that it wasn't always firing the progression event, at that point it checked the caseDetailsBefore to see if anything had changed, but we then removed that logic. Not sure why it is as a todo rather than just a comment though

Reply:
Its a TODO because I want to refactor when we call the progression methods. We will use caseDataBefore to check states. This was transisent and I didn't want to take it out.